zuul/layout.yaml

```diff
@@ -80,3 +80,7 @@ projects:
   - name: openstack/ironic
     experimental:
       - gate-ironic-dsvm-tempest-smoke-ironic-cimc-current-centos-7-cimc
+
+  - name: openstack/cinder
+    experimental:
+      - gate-cinder-dsvm-tempest-smoke-zm_cinder-current-cinder-node
```

Comment:
For consistency with the other jobs I would prefer if the job name "zm_cinder" was "zm-cinder" using "-" instead of "_"